tests/sentry/models/test_exporteddata.py

```diff
@@ -65,3 +89,32 @@ def test_finalize_upload(self):
         # Ensure the first file is deleted
         assert not File.objects.filter(id=self.file1.id).exists()
         assert self.data_export.date_expired == self.data_export.date_finished + timedelta(weeks=2)
+
+    def test_email_user(self):
+        # Shouldn't send if ExportedData is incomplete
+        with self.tasks():
+            self.data_export.email_user()
+        assert len(mail.outbox) == 0
+        # Should send one email if complete
+        self.data_export.finalize_upload(file=self.file1)
+        with self.tasks():
+            self.data_export.email_user()
+        assert len(mail.outbox) == 1
+
+    @patch("sentry.utils.email.MessageBuilder")
+    def test_email_user_content(self, builder):
```

Comment:
Saw this implemented somewhere else, and it seems like a good way of checking arguments